public/index.html

```diff
@@ -2,7 +2,6 @@
 <html lang="en">
```

Comment:
**[OPTIONAL]**

- [ ] It would be very nice if you make all your commit messages capitalized to make your pull request professional :+1: